src/ui.js

```diff
@@ -502,6 +502,18 @@ UIMode.Command = class extends UIMode {
     }
 };
 
+// We are viewing a diagram embedded in another page.
+UIMode.Embed = class extends UIMode {
```

Comment:
When we're in this mode, we also want to disable any keyboard shortcuts (e.g. currently the shortcut to export to LaTeX still works even in embed mode). Probably the easiest place to do this is somewhere in `handle_shortcut`, e.g. just returning if `ui.in_mode(UIMode.Embed)`.

Reply:
I'd also prefer to disable scrolling with the mouse wheel in embedded mode for now, here:
https://github.com/varkor/quiver/blob/b8c1ba4333fc584b7c7ec062ab9fabb4d88f9a58/src/ui.js#L955-L967